tests/jenkins/run_test_ubuntu.sh

```diff
@@ -19,7 +19,7 @@ fi
 cp make/config.mk .
 echo "USE_CUDA=1" >> config.mk
 echo "USE_CUDA_PATH=/usr/local/cuda" >> config.mk
-echo "USE_CUDNN=1" >> config.mk
+echo "USE_CUDNN=0" >> config.mk
```

Comment:
One the the jenkins scripts should build organic GPU operators rather than CUDNN always